ldap.example.yml

```diff
@@ -0,0 +1,14 @@
+address: link-of-ldap-server
+port:
+use_tls:
+base_dn:
+bind_username:
+bind_password:
+search_class:
+username_attribute:
+full_name_attribute:
```

Comment:
I seriously doubt we need all these attributes. Just the `username` one seems to suit all our needs.

Reply:
Έφυγαν το fullname, attribute και fix_full_name.